src/string.rs

```diff
@@ -6,6 +6,7 @@ use generic_array::ArrayLength;
 use Vec;
 
 /// A fixed capacity [`String`](https://doc.rust-lang.org/std/string/struct.String.html)
+#[derive(Clone)]
```

Comment:
This should be a manual `Clone` implementation. No need for a `Clone` bound on `N`.